src/pages/jobs.js

```diff
@@ -0,0 +1,160 @@
+import { useState } from 'react'
+import Form from '../components/Jobs/Form/Form'
+import Card from '../components/Jobs/Card/Card'
+import Loader from '../components/Jobs/Loader/Loader'
+import Video from '../components/Jobs/Video/Video'
+import Pagination from '../components/Pagination/Pagination'
+import PageHeader from '../components/PageHeader'
+import { StaticImage } from 'gatsby-plugin-image'
+import SEO from '../components/SEO'
+
+function Jobs() {
+  const [jobData, setJobData] = useState(false)
+  const [formSubmitEvent, setFormSubmitEvent] = useState(false)
+  const [formSubmitted, setFormSubmitted] = useState(false)
+  const [pageContext, setPageContext] = useState(false)
+
+  function formData(formResponse, page = 1) {
+    setFormSubmitEvent(formResponse)
+    setFormSubmitted(true)
+    document.getElementById('scroll-to').scrollIntoView({ behavior: 'smooth', block: 'start' })
+
+    let what = 'javascript Reactjs gatsby graphql nodejs node.js jquery bootstrap'
```

Comment:
Use `const` over `let`

Reply:
When using symbolic constants usually the variable name is all caps. `const WHAT = 'javascript Reactjs gatsby graphql nodejs node.js jquery bootstrap'`